exercises.ipynb

```diff
@@ -172,22 +287,131 @@
     "\n",
     "HINT: Write a separate function to get the current exchange rates! That can be reused!"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_exchange_rates(date='latest', base='EUR'):\n",
```

Comment:
Nice default values! In general, I would try to put them in only one function, otherwise it can be hard to keep them in sync!